srcs/parsing/lexer.c

```diff
@@ -88,10 +111,19 @@ void	get_command_tables(t_cmd_table *cmd_table, int nb_cmds, char **instructions
 		if (cmd_table[i].nb_redirs)
 		{
 			cmd_table[i].redirs = get_redirs(instructions[i], cmd_table[i].nb_redirs);
-			temp = rewrite_instruction_without_redirs(instructions[i]);
-			free(instructions[i]);
-			instructions[i] = temp;
+			j = 0;
+			while (j < cmd_table[i].nb_redirs)
+			{
+				if (is_empty(cmd_table[i].redirs[j].arg))
+					error_and_return(REDIR_NO_ARG, SYNTAX_ERROR);

```

Comment:
But what happens now when you run ls | >>
(or whatever that leads to this error)?
Cos the way I see it we need to stop it so to speak. Right now it just keeps on going.
And update minishell->exit_code.